v2.0/src/Components/Pages/ProfilePage/AddingHouseholdForm.js

```diff
@@ -142,6 +142,7 @@ class AddingHouseholdForm extends React.Component {
           <METextField
             type="text"
             name="zip"
+            required={true}
```

Comment:
Hi @BradHN1 ,if you intend to make zip code compulsory, `required = {true}` isnt doing the job. You might have to put an extra check in the onsubmit statement to check if a user input something.... Or, I can investigate and fix this later.

Reply:
Hi @frimpongopoku - can you check out the changes I made...  (which go along with changes to API